client/wordpress-com.js

```diff
@@ -198,121 +191,110 @@ const sections = [
 	}
 ];
 
-editorPaths = [ '/post', '/page' ];
-if ( config.isEnabled( 'manage/custom-post-types' ) ) {
-	editorPaths.push( '/edit' );
-}
-
 sections.push( {
 	name: 'post-editor',
-	paths: editorPaths,
+	paths: [ '/post', '/page', '/edit' ],
 	module: 'post-editor',
 	group: 'editor',
 	secondary: true
 } );
 
-if ( config.isEnabled( 'account-recovery' ) ) {
-	sections.push( {
-		name: 'account-recovery',
-		paths: [ '/account-recovery' ],
-		module: 'account-recovery',
-		secondary: false,
-		enableLoggedOut: true,
-	} );
-}
+sections.push( {
+	name: 'account-recovery',
+	paths: [ '/account-recovery' ],
+	module: 'account-recovery',
+	secondary: false,
+	enableLoggedOut: true,
+} );
 
-if ( config.isEnabled( 'manage/drafts' ) ) {
-	sections.push( {
-		name: 'posts-pages',
-		paths: [ '/drafts' ],
-		module: 'my-sites/drafts',
-		secondary: true,
-		group: 'sites'
-	} );
-}
+sections.push( {
```

Comment:
I looked into it, and its already there :) 
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/drafts/index.js